<?xml version="1.0"?>
<launch>
	<group ns="planning" >
		<rosparam command="load" file="$(find drone_path_planning)/config/prob_definitiion.yaml" />
	</group>
	
	<node name="path_planner" pkg="drone_path_planning" type="rigidBodyPath.py" output="screen"/>

  	<include file="$(find execution)/launch/Rviz.launch">
	</include>

	
</launch>
